local/compose/create.go

```diff
@@ -94,6 +94,10 @@ func getContainerCreateOptions(p *types.Project, s types.ServiceConfig, number i
 		containerNumberLabel: strconv.Itoa(number),
 	}
 
+	for k, v := range s.Labels {
```

Comment:
doing so, compose file can override `com.docker.compose.xx` labels. Sounds crazy, but docker-compose prevent this. I can't imagine any use-case to require this, but users are full of creativity.

Reply:
Oh ok, so `docker-compose` silently rewrites the label, ok will fix